### ISSUE TITLE
reduce the amalgamated PyNE

### ISSUE THREAD

**Issue by kkiesling** [svalinn/DAGMC#420]
We need such a small fraction of PyNE, we can either amalgamate our own PyNE or strip down the PyNE-provided amalgamation.

**Comment by makeclean** [comment]
Sure think, I now regret including decay.cpp.

**Comment by makeclean** [comment]
The minimal set is here, https://github.com/svalinn/DAGMC/blob/develop/pyne/README.md, just remove decay.cpp and decay.h

**Comment by gonuke** [comment]
That looks like the current set - not the minimal set.

**Comment by gonuke** [comment]
You don't need `enrichment*` either..

And `json*` ??

**Comment by gonuke** [comment]
It's the enrichment stuff that takes all the space

**Comment by gonuke** [comment]
```
ls -lSr enrichment*
-rw-rw-r-- 1 wilsonp wilsonp      37 Feb  3 20:27 enrichment_symbolic.cpp
-rw-rw-r-- 1 wilsonp wilsonp     778 Feb  3 20:27 enrichment_cascade.cpp
-rw-rw-r-- 1 wilsonp wilsonp    1067 Feb  3 20:27 enrichment_symbolic.h
-rw-rw-r-- 1 wilsonp wilsonp    1715 Feb  3 20:27 enrichment_cascade.h
-rw-rw-r-- 1 wilsonp wilsonp    7519 Feb  3 20:27 enrichment.h
-rw-rw-r-- 1 wilsonp wilsonp   16647 Feb  3 20:27 enrichment.cpp
-rw-rw-r-- 1 wilsonp wilsonp   46076 Feb  3 20:27 enrichment_symbolic05.cpp
-rw-rw-r-- 1 wilsonp wilsonp  166401 Feb  3 20:27 enrichment_symbolic10.cpp
-rw-rw-r-- 1 wilsonp wilsonp  381665 Feb  3 20:27 enrichment_symbolic15.cpp
-rw-rw-r-- 1 wilsonp wilsonp  695403 Feb  3 20:27 enrichment_symbolic20.cpp
-rw-rw-r-- 1 wilsonp wilsonp 1460061 Feb  3 20:27 enrichment_symbolic30.cpp
-rw-rw-r-- 1 wilsonp wilsonp 2551882 Feb  3 20:27 enrichment_symbolic40.cpp
```

**Comment by gonuke** [comment]
Oh... and now that I ran `decaygen.py` I see 16MB of decay.... :(

**Comment by gonuke** [comment]
Maybe we can also leave out:
* data
* rxname

**Comment by makeclean** [comment]
We need json for the encoding and decoding of material objects I think

**Comment by makeclean** [comment]
We also nee data for the inclusion of atomic_data

**Comment by makeclean** [comment]
So probably, the removal of enrichment* and decay* would be the minimum working set.

**Comment by makeclean** [comment]
So I will revise my earlier statement, https://github.com/svalinn/DAGMC/blob/develop/pyne/README.md, minus enrichment* and decay*, and thats probably it.

**Comment by makeclean** [comment]
Yeah, json definitely needed by material, and atomic_data included by data.

**Comment by ljacobson64** [comment]
Closing this since I think it was addressed in [PR#427](https://github.com/svalinn/DAGMC/pull/427)
